easyInterface/Diffraction/DataClasses/Utils/BaseClasses.py

```diff
@@ -1,10 +1,12 @@
+import numpy as np
 from copy import deepcopy
 from typing import Union, Optional, Any, NoReturn
 
 from easyInterface.Utils.units import Unit
 from easyInterface.Utils.DictTools import PathDict, UndoableDict
+from easyInterface import logger
 from easyInterface.Utils.Logging import logging
-from easyInterface import VERBOSE
+#from easyInterface import VERBOSE
```

Comment:
Can be replaced with:
`from easyInterface import logger, VERBOSE`
I do not see logging being used.

Reply:
Debug logging is removed